packages/core/test/logger.js

```diff
@@ -335,6 +335,39 @@ describe('logger', () => {
     ]);
   });
 
+  it('should leave response content of null uncensored', async () => {
+    const event = {
+      method: 'authentication.sessionConfig.perform',
+    };
+    const logger = createlogger(event, options);
+
+    const { message, data } = prepareTestRequest({
+      resBody: JSON.stringify(null),
+    });
+
+    logger(message, data);
```

Comment:
I removed `|| {}`, trying to reproduce the same error:

```
TypeError: Cannot convert undefined or null to object
```

I found that we need to add an `await` here. Otherwise, the error would be silently swallowed.

```suggestion
    await logger(message, data);
```

Reply:
Thanks. I updated it @eliangcs